examples/offline-analysis-example.py

```diff
@@ -16,6 +16,8 @@
     # Initialize a 3G/4G monitor
     src = OfflineReplayer()
     src.set_input_path("./offline_log_example.mi2log")
+    # src.set_input_path("./demo/RRCStateMachine/LTE_RRC_state_sample.mi2log")
```

Comment:
Please remove these two lines since they do not belong to the official example case